runtime/oti/j9nonbuilder.h

```diff
@@ -5657,6 +5721,12 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t jclCacheMutex;
 	UDATA arrayletLeafSize;
 	UDATA arrayletLeafLogSize;
+	UDATA contiguousIndexableHeaderSize;
+	UDATA discontiguousIndexableHeaderSize;
+	UDATA isIndexableDataAddrPresent;
```

Comment:
`BOOLEAN` might be a better type for this field. Why is it not conditional on 64-bit builds (i.e. after line 5727)?

Reply:
Yes I agree. It used to be a `BOOLEAN` and also conditional on 64-bit builds, but I was getting some DDR failures when this was the case. I will go back to this again and confirm why this was done.

Reply:
You'll need to deal with the possibility that the field is unknown anyway (in old core files, or from 32-bit VMs).

Reply:
I have tested current changes (`isIndexableDataAddrPresent` is now conditional on 64-bit builds, however, it remains as type `UDATA` as setting it to `BOOLEAN` leads to a compile error on Java 8 or earlier - http://vmfarm.rtp.raleigh.ibm.com/job_output.php?id=47471003)  with a core generated from master branches and there are no failures (if `isIndexableDataAddrPresent` field is not present, the correct legacy values are used to handle this case).